docs/basic-workflow.md

```diff
@@ -0,0 +1,405 @@
+8:40:23AM:  L ok: waiting on replicaset/simple-app-549958d5dd (apps/v1) namespace: default
+8:40:23AM:  L ongoing: waiting on pod/simple-app-549958d5dd-4sfbz (v1) namespace: default
+8:40:23AM:     ^ Pending: ContainerCreating
+8:40:24AM: ok: reconcile deployment/simple-app (apps/v1) namespace: default
+8:40:24AM: ---- applying complete [2/2 done] ----
+8:40:24AM: ---- waiting complete [2/2 done] ----
+
+Succeeded
+```
+
+You have successfully deployed an OCI image to Kubernetes using `imgpkg` and `kapp`. Before continuing, 
+remove the application from your cluster:
+
+```shell
+kapp delete -a simple-app -y
+```
+
+### Bundle distribution
+
+For more information on bundles, read more [here](README.md#images-vs-bundles).
+
+#### How to distribute the bundle
+
+In the folder [examples/basic-bundle](../examples/basic-bundle), there is a set of configuration files that
+will allow a user to create a service and a deployment for an application that will run on Kubernetes. The 
+folder also contains the [`.imgpkg`](resources.md#imageslock) hidden directory with an `ImagesLock` file and 
+a [bundle metadata file](resources.md#bundle-metadata).
+
+```shell
+examples/basic-bundle
+├── .imgpkg
+│   ├── bundle.yml
+│   └── images.yml
+└── config.yml
+```
+
+You can push the above folder containing a bundle to your local Docker registry using the following command:
+
+`imgpkg push -f examples/basic-bundle -b localhost:5000/simple-app-bundle`
+
+Flags used in the command:
+  * `-f` indicates the folder the user want to package
+  * `-b` indicates that the user want to push a bundle to the registry
+
+The output displays all the files that will be packaged and the destination of the bundle:
+```shell
+dir: .
+dir: .imgpkg
+file: .imgpkg/bundle.yml
+file: .imgpkg/images.yml
+file: config.yml
+Pushed 'localhost:5000/simple-app-bundle@sha256:ec3f870e958e404476b9ec67f28c598fa8f00f819b8ae05ee80d51bac9f35f5d'
+Succeeded
+```
+
+#### How to retrieve the bundle
+
+You can retrieve the bundle by running the following command to download the bundle:
+
+`imgpkg pull -o /tmp/simple-app-bundle -b localhost:5000/simple-app-bundle`
+
+Flags used in the command:
+  * `-o` indicates the folder where the bundle will be unpacked
+  * `-b` indicates that the user want to pull a bundle from the registry
+
+The output shows the image pull was successful:
+```shell
+Pulling image 'localhost:5000/simple-app-bundle@sha256:ec3f870e958e404476b9ec67f28c598fa8f00f819b8ae05ee80d51bac9f35f5d'
+Extracting layer 'sha256:7906b9650be657359ead106e354f2728e16c8f317e1d87f72b05b5c5ec3d89cc' (1/1)
+Locating image lock file images...
+One or more images not found in bundle repo; skipping lock file update
+
+Succeeded
+```
+
+__Note:__ The message `One or more images not found in bundle repo; skipping lock file update` indicates that the file 
+`/tmp/simple-app-bundle/.imgpkg/images.yml` was not updated since the bundle has never been pushed to the Docker registry 
```

Comment:
```suggestion
`/tmp/simple-app-bundle/.imgpkg/images.yml` was not updated since the bundle has never been copied to a different Docker registry 
```